wiki/Performance_Troubleshooting/vi.md

```diff
@@ -0,0 +1,106 @@
+- Đảm bảo bạn đang bật [Game mode](https://quantrimang.com/huong-dan-kich-hoat-che-do-game-mode-tren-windows-10-130041). Nó cho phép Windows cung cấp nhân (CPU/GPU) và tài nguyên dành riêng cho osu!.
+- Mở Task manager, chuyển sang tab Details và sắp xếp danh sách theo thứ tự tiêu tốn CPU. Kiểm tra xem mỗi khi xảy ra đứng khung hình, có ứng dụng nào tiêu tốn CPU nhiều không, như virus chạy dưới nền, hoặc trình quét diệt virus. Bạn có thể chạy osu! ở chế độ cửa sổ, hoặc để Task Manager ở màn hình thứ hai để chẩn đoán hiện tượng khi đang chơi hoặc chạy beatmap với mod Tự động.
+
+## Giới hạn khung hình
+
+osu! cho bạn nhiều lựa chọn giới hạn khung hình. Chúng đều có ưu và nhược điểm riêng. Chú ý rằng ở bất cứ thiết đặt nào, osu! sẽ giới hạn khung hình ở menu để hạn chế việc sử dụng CPU.
+
+### VSync (Đồng bộ dọc)
+
+Mức giới hạn này phụ thuộc vào cơ chế đồng bộ dọc của driver. Nó đảm bảo không có hiện tượng xé hình, nhưng sẽ làm tăng độ trễ khoảng 1-2 khung hình vì cần phải buffer khung hình trước khi hiển thị.
+
+Không khuyến nghị cho màn hình 60Hz. Nhưng nó có thể hữu ích hơn ở màn hình trên 120Hz, nhưng với màn hình có GSync hay FreeSync, thì việc dùng VSync để tránh xé hình không còn cần thiết nữa.
+
+### Power Saving (Tiết kiệm điện)
+
+Mức giới hạn này nhắm tới việc tiết kiệm điện trong khi vẫn đáp ứng được trải nghiệm game ổn định. Game sẽ chạy với fps bằng 2 lần tần số quét của màn hình và cố gắng giữ ổn định tần suất khung hình.
+
+Khuyến nghị cho máy tính cổ hoặc laptop cần tiết kiệm pin.
+
+### Optimal (Tối ưu)
+
+Mức giới hạn này giữ tần suất khung hình bằng 8 lần tần số quét màn hình, tối đa là 960 FPS. Đây là mức giới hạn được khuyến nghị cho người dùng không muốn đốt CPU/GPU quá mức, nhưng vẫn đảm bảo hiệu năng đủ cao dẫn đến độ trễ tín hiệu vào thấp.
+
+Thường được ưa chuộng hơn Unlimited để làm giảm số khung hình được xuẩt ra, đồng thời làm giảm khả năng dẫn đến [garbage collection](https://en.wikipedia.org/wiki/Garbage_collection_(computer_science) "Wikipedia") overhead (thứ dẫn đến khung hình bị đứng).
+
+### Unlimited (Không giới hạn)
+
+Đây là lựa chọn không làm giới hạn tần suất khung hình. osu! sẽ xuất hình nhanh hết mức mà CPU/GPU có thể.
+
+Nghe thì có vẻ là lựa chọn tốt nhất (và đúng là như vậy, nếu phần cứng bạn đáp ứng được), thì có vài điều bạn cần cân nhắc:
+
+- Bằng việc xuất nhiều khung hình hơn, có nhiều khả năng dẫn đến [garbage collection](https://en.wikipedia.org/wiki/Garbage_collection_(computer_science) "Wikipedia") overhead.
+- Bằng việc đẩy CPU và GPU đến cực hạn, có khả năng chúng sẽ nghẽn, dẫn đến giảm hiệu năng tổng thể.
+- GPU vốn dĩ không dùng để xuất hơn 1000 khung hình trên giây, nên nhiều hành vi bất thường có thể xảy ra, bao gồm giảm hiệu năng và tăng độ trễ thay vì giảm đi theo lý thuyết.
+
+Tuy nhiên, nếu sự sinh nhiệt và tiềm năng làm phần cứng bạn quá tải không phải là vấn đề đáng quan tâm, Unlimited có thể là lựa chọn đem lại tần suất frame ổn định nhất.
```

Comment:
```suggestion
Tuy nhiên, nếu sự sinh nhiệt và nguy cơ làm phần cứng của bạn quá tải không phải là vấn đề đáng quan tâm, Unlimited có thể là lựa chọn đem lại tần suất khung hình ổn định nhất.
```

Reply:
I'll change the first twos, for the third, I'll change `tần suất frame` to `FPS`.